OpenRA.Mods.Common/Orders/GuardOrderGenerator.cs

```diff
@@ -28,29 +28,30 @@ protected override IEnumerable<Order> OrderInner(World world, CPos xy, MouseInpu
 				yield break;
 
 			var target = FriendlyGuardableUnits(world, mi).FirstOrDefault();
-			if (target == null || Subjects.All(s => s.IsDead))
```

Comment:
Are you 100% sure that actors in `Subjects` can't die between `SelectionChanged` being called and this? We haven't ever checked this in `AttackMoveOrderGenerator` so I assume the answer is yes, but it would be good to get confirmation that you have thought about this.

Reply:
There was a reason for this change and I think actors can't die in-between, but don't remember anymore.